### ISSUE TITLE
使用注解方式配置服务，怎么对Service里面的方法配置retries？

### ISSUE THREAD

**Issue by xiaobzhou** [apache/incubator-dubbo#2324]
使用注解方式配置服务，怎么对Service里面的方法配置retries？基于xml的方式采用<dubbo:method/>

**Comment by dengjiexiao** [comment]
我想问下dubbo admin 怎么看不到消费者了？

**Comment by kimmking** [comment]
@xiaobzhou I think we miss an annotation for method config.

**Comment by kimmking** [comment]
@dengjiexiao  maybe a new issue is good for our discussion.

**Comment by beiwei30** [comment]
agree, we need to align annotation support with xml config, right now, it is insufficient to only have **@reference** and **@service**
